scripts/configure-venue-access.ts

```diff
@@ -94,16 +74,28 @@ initFirebaseAdminApp(projectId);
       console.log(
         `Setting venues/${venueId}/access/${method} to {emails: ${emails}}...`
       );
-      await accessDocRef.set({
-        emails: mergeStringArrays(emails, (access as VenueAccessEmails).emails),
-      });
+      await accessDocRef.set(
+        {
+          emails: emails,
+        },
+        { merge: true }
+      );
```

Comment:
Will this merge `emails` into the existing `emails` in firebase? Or will it overwrite that key entirely (but not wipe out any of the other data)?

If the former, awesome, but if not, I was thinking that `arrayUnion` would probably be more what we want to use here:

- https://firebase.google.com/docs/firestore/manage-data/add-data#update_elements_in_an_array

Reply:
> Will this merge emails into the existing emails in firebase?

Yes, it merges them.

Reply:
Awesome!